index.js

```diff
@@ -2787,64 +2787,62 @@
   }
   S.prop = def('prop', {a: [Accessible]}, [$.String, a, b], prop);
 
-  //# get :: Accessible a => TypeRep b -> String -> a -> Maybe b
+  //# get :: Accessible a => (b -> Boolean) -> String -> a -> Maybe c
   //.
-  //. Takes a [type representative](#type-representatives), a property
-  //. name, and an object and returns Just the value of the specified object
-  //. property if it is of the specified type (according to [`is`](#is));
-  //. Nothing otherwise.
-  //.
-  //. The `Object` type representative may be used as a catch-all since most
-  //. values have `Object.prototype` in their prototype chains.
+  //. Takes a predicate, a property name, and an object and returns Just the
+  //. value of the specified object property if it exists and the value
```

Comment:
This seems slightly ambiguous to me. I see two valid interpretations:

  - _return Just (the value of the specified object property) if it exists_; or
  - _return Just the value of (the specified object property) if it exists_.

"It" can refer to either parenthesized term, depending on where one's brain decides the parens should go. For some reason my brain wants to include "the value of" in the parens.